components/brave_rewards/core/contribution/contribution_external_wallet.cc

```diff
@@ -105,14 +105,6 @@ void ContributionExternalWallet::ContributionInfo(
   callback(mojom::Result::LEDGER_OK);
 }
 
-void ContributionExternalWallet::OnSavePendingContribution(
-    const mojom::Result result) {
-  if (result != mojom::Result::LEDGER_OK) {
-    BLOG(0, "Problem saving pending");
-  }
-  ledger_->client()->PendingContributionSaved(result);
```

Comment:
`PendingContributionSaved()` can now be removed from `ledger.mojom.LedgerClient`, also from `RewardsServiceImpl`, and test/mock ledger client.

Reply:
And on iOS.

Reply:
Issue created here: https://github.com/brave/brave-browser/issues/29694